main.js

```diff
@@ -0,0 +1,76 @@
+let books = [];
+
+const Books = function (title, author) {
+  this.title = title;
+  this.author = author;
+};
+
+// Select the Elements
+const container = document.querySelector('.books-container');
+const addBtn = document.querySelector('form');
+
+const titleInput = document.querySelector('.title-input');
+const authorInput = document.querySelector('.author-input');
+
+// Data Storage
+const storage = function (books) {
+  localStorage.setItem('books', JSON.stringify(books));
+};
+
+// Add Book
+const addBook = function (title, author) {
+  const newBook = new Books(title, author);
+  books.push(newBook);
+  storage(books);
+};
+
+// Remove Book
+const removeBook = function (title) {
+  books = books.filter((book) => book.title !== title);
+  storage(books);
+};
```

Comment:
- [ ] So far so good. It would have been better to add an `id` property to a book object which should be unique. The `id` property could then be used to filter books. Since the `id` property is unique, only one book(**_`the correct/intended book`_**) will be deleted at a time.
  
  Right now when I add books with the same `title` and delete only one of them, the rest of the same-title books will also be deleted, this is evident on page refresh and also in the localStorage. Therefore, we will need to refactor the removeBook method to ensure that the following project requirement is achieved: 

---
>  - Make sure that when a user clicks the "Remove" button:
>     - The correct book is removed from the collection.
>     - The correct book disappears from the page.
---

Another solution would be to prevent the addition of duplicate books in the `addBook` method.

Please work on this.

Reply:
Hi @Lameck1 👋
Thank you for your feedback 🙏.

I will prevent the addition of duplicate books in the `addBook` method As you suggested to me. 👌
If I implement this solution I don't need to add `id`, Right?
And also, think no two books have the same `title`, so the `title` can be unique instead of I add `id`.
Is this ok?

Reply:
Perfectly fine